src/table.js

```diff
@@ -1322,6 +1322,72 @@ Table.prototype.sampleRowKeysStream = function(gaxOptions) {
   ]);
 };
 
+/**
+ * Returns true in callback if consitency check is true for
+ * generated consistency token.
+ * In-case consistency check returns false, retrial is done in interval
+ * of 5 seconds till 10 minutes, after that it returns false.
+ *
+ * @param {function(?Error, ?boolean)} [callback]
+ * The function which will be called with the result of the API call.
+ */
+Table.prototype.waitForReplication = function(callback) {
```

Comment:
We should expose the `gaxOpts` argument to this method, to be consistent with the other methods.

Reply:
Could not understand, 
If we are not using gaxOptions in this method, the why we should add this as  an argument to this method?